python/install.py

```diff
@@ -7,6 +7,9 @@
 src_root = sys.argv[1]
 target_prefix = sys.argv[2]
 install = int(sys.argv[3])
+target_destdir = os.environ.get("DESTDIR", "")
```

Comment:
An attempt has been made support this behavior with the cmake variable `${DEST_PREFIX}`[here](https://github.com/OPM/opm-common/blob/master/CMakeLists.txt#L314) - do we now have two ways to (try) to achieve the same now?

Reply:
i'm not at all sure what a DEST_PREFIX at configure time is supposed to be used for.
it is purely a install time construct, used by packaging systems to install into a temporary dir.

my solution is the correct one.

Reply:
OK;

the way it was meant to be used (and I think that has worked ...) was:
```txt
bash% cmake ..  -DDEST_PREFIX=/path/to/install/root
bash% make
bash% make install
```
then `DDEST_PREFIX` would be respected by the full `make install` target.

Reply:
well yes, but as i said that's configure time. i would have to guess what the destdir would be cause it is only provided by the dpkg tools at installation time through the env var.